release/release_gold.sh

```diff
@@ -183,6 +190,19 @@ function promote_packages {
 	ssh root@lrdcom-vm-1 cp -a "/www/releases.liferay.com/${LIFERAY_RELEASE_PRODUCT_NAME}/release-candidates/${_ARTIFACT_RC_VERSION}" "/www/releases.liferay.com/${LIFERAY_RELEASE_PRODUCT_NAME}/${_PRODUCT_VERSION}"
 }
 
+function get_latest_product_version {
+	if [ -e "releases.json" ]
+	then
+		rm -fr releases.json
+	fi
+	
+	lc_download "https://releases.liferay.com/releases.json" releases.json >/dev/null
```

Comment:
@kiwm, can we use `&> /dev/null` instead?

Additionally, please make sure that if an error happens during the execution of the download of this file, the error message will still be displayed. Tip: simulate an error using `https://releases.liferay.com/releasesaaaa.json`.